packages/admin/src/FilamentServiceProvider.php

```diff
@@ -98,6 +98,8 @@ protected function bootLivewireComponents(): void
             $this->app->singleton(Component::class, fn () => $component);
```

Comment:
We can get rid of this binding now AFAIK

Reply:
✅